docs/versioned_docs/version-7.4.x/configuration/auth.md

```diff
@@ -136,6 +136,7 @@ Note: When using the ADFS Auth provider with nginx and the cookie session store
 
 1.  Create a new project: https://github.com/settings/developers
 2.  Under `Authorization callback URL` enter the correct url ie `https://internal.yourcompany.com/oauth2/callback`
+3.  Configure an explicit scope, e.g. with `--scope="user:email"` ([issue 1724](https://github.com/oauth2-proxy/oauth2-proxy/issues/1724))
```

Comment:
We need to add this to `docs/docs` which is the master copy of the documentation, until the bug is fixed at least

Reply:
@JoelSpeed @Sanqui as far as I understand it, the issue will not be resolved by setting the scope via the flag. The issue is that the scope will only be set when using a config file.